opsdroid/connector/matrix/connector.py

```diff
@@ -373,6 +373,25 @@ def get_roomname(self, room):
 
         return room
 
+    async def joined_rooms(self, encrypted_only=False):
+        """Returns a list of room_id of all joined rooms"""
+
+        room_ids = await self.connection.joined_rooms()
+
+        if isinstance(room_ids, nio.responses.JoinedRoomsError):
+            _LOGGER.error(
+                f"Error while getting list of joined rooms from room state: {room_ids.message} (status code {room_ids.status_code})"
+            )
+            return
+        else:
```

Comment:
You also don't need this else: block as this code will not be executed if the early return happens. So you can dedent this section.

Reply:
Yes. Fine I will change it